src/main/resources/bootstrap.yaml

```diff
@@ -5,13 +5,22 @@ spring:
       paths: /mnt/secrets/bulk-scan
       aliases:
         processor-POSTGRES-PASS: BULK_SCANNING_DB_PASSWORD
+        processor-staging-db-password: BULK_SCANNING_DB_PASSWORD
+        processor-staging-db-user: BULK_SCANNING_DB_USER_NAME
+        processor-staging-db-name: BULK_SCANNING_DB_NAME
+        processor-staging-db-host: BULK_SCANNING_DB_HOST
+        processor-staging-db-port: BULK_SCANNING_DB_PORT
```

Comment:
How is it that these mappings are not defined for non-staging DB?

Reply:
good point @lgonczar they are defined in values yaml `https://github.com/hmcts/bulk-scan-processor/blob/master/charts/bulk-scan-processor/values.yaml#L27-L30` so for staging we can do similar in value aat template